### PR TITLE
Make this plugin work with the PCT

### PR DESCRIPTION
**Pull request by recena**
With these changes this plugin can be used in the [PCT](https://github.com/jenkinsci/plugin-compat-tester).

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,9 +14,9 @@
   <packaging>hpi</packaging>
 
   <scm>
-    <connection>scm:git:ssh://github.com/jenkinsci/kubernetes-credentials-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/kubernetes-credentials-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/kubernetes-credentials-plugin</url>
+    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <developerConnection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</developerConnection>
```
Comment (Vlatombe) [inline pom.xml]: Looks incorrect, `git://` is a readonly protocol. Shouldn't it be `ssh://git@` instead?

Comment (recena) [reply pom.xml]: I'm using what we are using in [other plugins](https://github.com/jenkinsci/github-branch-source-plugin/blob/master/pom.xml#L29).

Comment (Vlatombe) [reply pom.xml]: for connection, it's fine, but for developerConnection, it's not. The plugin you reference has `git@github.com` in developerConnection string which is implicitly ssh.

Comment (recena) [reply pom.xml]:
```
    <scm>
        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
        <tag>HEAD</tag>
    </scm>
```

**Pull request merged**